package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need to check in this?

Reply:
This is necessary for local site checking. If that is unlikely to happen, we can remove it and the site can be checked exclusively at the PR stage.